### PR TITLE
Refactor Selectors for the footer

### PR DESCRIPTION
**Pull request by Nifargo**
Selectors for the Footers

### CODE REVIEW

**Comment by dashasandler** [comment]
Looks good, but strange name "Andrw" what does it mean?

**Review by dashasandler — APPROVED** [review]

**Comment by MariaMoroz** [comment]
Change, please, the name of PR

**Comment by SergeiKuznetsov** [comment]
The name of the PR will go into the header of the squash commit message, and you have a meaningless set of letters there.

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "@wdio/local-runner": "^7.5.7",
     "@wdio/mocha-framework": "^7.5.3",
     "@wdio/spec-reporter": "^7.5.7",
+    "@wdio/sync": "^7.7.3",
```
Comment (SamJulia) [inline package.json:27]: you do not need to change package.json file

**Review by SamJulia — CHANGES_REQUESTED** [review]
Does't need to update package.js

**Comment by SergeiKuznetsov** [comment]
you do not need to delete package.json file

**Comment by SergeiKuznetsov** [comment]
@SamJulia tests are unstable. 
Is it because everyone runs under the same user?

**Pull request closed without merging**